doc/source/how-to-enable-ssl-connections.rst

```diff
@@ -39,40 +39,80 @@ using the scripts mentioned in this guide.
 Server (SuperLink)
 ------------------
 
-Use the following terminal command to start a sever (SuperLink) that uses the previously
-generated certificates:
+Navigate to ``examples/advanced-tensorflow`` and use the following terminal command to
+start a server (SuperLink) that uses the previously generated certificates:
 
 .. code-block:: bash
 
-    flower-superlink
-      --ssl-ca-certfile certificates/ca.crt
-      --ssl-certfile certificates/server.pem
-      --ssl-keyfile certificates/server.key
+    $ flower-superlink \
+        --ssl-ca-certfile .cache/certificates/ca.crt \
+        --ssl-certfile .cache/certificates/server.pem \
+        --ssl-keyfile .cache/certificates/server.key
 
 When providing certificates, the server expects a tuple of three certificates paths: CA
 certificate, server certificate and server private key.
 
-Client (SuperNode)
-------------------
+Clients (SuperNode)
+-------------------
 
 Use the following terminal command to start a client (SuperNode) that uses the
 previously generated certificates:
 
 .. code-block:: bash
 
-    flower-supernode
-         --root-certificates certificates/ca.crt
-         --superlink 127.0.0.1:9092
+    $ flower-supernode \
+        --root-certificates .cache/certificates/ca.crt \
+        --superlink 127.0.0.1:9092 \
+        --clientappio-api-address 0.0.0.0:9095 \
+        --node-config="partition-id=0 num-partitions=10"
 
 When setting ``root_certificates``, the client expects a file path to PEM-encoded root
 certificates.
 
+In another terminal, start a second SuperNode that uses the same certificates:
+
+.. code-block:: bash
+
+    $ flower-supernode \
+        --root-certificates .cache/certificates/ca.crt \
+        --superlink 127.0.0.1:9092 \
+        --clientappio-api-address 0.0.0.0:9096 \
+        --node-config="partition-id=1 num-partitions=10"
+
+Note that in the second SuperNode, you must specify a different port for the
+``ClientAppIO`` API address to avoid clashing with the first SuperNode.
```

Comment:
```suggestion
Note that in the second SuperNode, if you run both on the same machine, you must specify a different port for the
``ClientAppIO`` API address to avoid clashing with the first SuperNode.
```